lib/ipmi_sensor.c

```diff
@@ -270,7 +288,36 @@ ipmi_sensor_print_fc_threshold(struct ipmi_intf *intf,
 		thresh_available = 0;
 
 	if (csv_output) {
-		/* NOT IMPLEMENTED */
+                printf("%s,", sr->s_id);
+        if (sr->s_reading_valid) {
+            if (sr->s_has_analog_value)
+                printf(",%.3f,%s,%s",
+                       sr->s_a_val, sr->s_a_units, thresh_status);
+            else
+                printf(",0x%x,%s,%s",
+                       sr->s_reading, sr->s_a_units, thresh_status);
+        } else {
+            printf(",%s,%s,%s",
+                   "na", sr->s_a_units, "na");
+        }
+        if (thresh_available && sr->full) {
+#define PTS(bit, dataidx) {                     \
+print_thresh_setting(sr->full, rsp->data[0] & (bit),        \
+    rsp->data[(dataidx)], ",", "%.3f", "0xx", "%s");    \
+}
+            PTS(LOWER_NON_RECOV_SPECIFIED,  3);
+            PTS(LOWER_CRIT_SPECIFIED,   2);
```

Comment:
Why so many spaces after the comma? Why different amount of them on each line? Must be exactly 1 everywhere.